<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MastersMenu</class>
 <widget class="QWidget" name="MastersMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Управление Мастерами</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
  </property>
  <widget class="QLabel" name="MainTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>132</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>Мастеры</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="FIOLayout">
    <property name="spacing">
     <number>65</number>
    </property>
    <item>
     <widget class="QLabel" name="FIOLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FIOEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="AddMaster">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить мастера</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveMaster">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>530</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить мастера</string>
   </property>
  </widget>
  <widget class="QTableView" name="MastersTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>771</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RegLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="RegLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Регистрационный номер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PositionLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="PositionLabel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="PositionCombo">
      <item>
       <property name="text">
        <string>Junior</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Middle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Senior</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
